### PR TITLE
Fix double space bug

### PR DESCRIPTION
**Pull request by romatik**
Yinghui found a case where sentence like `I bought a sturdy and  beautiful shelf.` would be parsed incorrectly

### CODE REVIEW

File: extra_model/_aspects.py [hunk 1]
```diff
@@ -65,12 +62,10 @@ def adjective_list(tokens):
...
-                grandchild
-            ) in (
-                child.children
-            ):  # find both X and Y in patterns of the form "the X and Y product"
-                if grandchild.dep_ == "conj":
+            for grandchild in child.children:
+                # find both X and Y in patterns of the form "the X and Y product"
+                if grandchild.dep_ == "conj" and not grandchild.is_space:
+                    # grandchild.is_space is handling double space cases
                     adjectives.append(grandchild.text)
```
Comment (SanRehmo) [inline extra_model/_aspects.py:69]: might make sense to move this to a generic helper function as this is basically duplicate code from line 43-49

File: extra_model/_aspects.py [hunk 2]
```diff
@@ -65,12 +62,10 @@ def adjective_list(tokens):
...
-            for (
-                grandchild
-            ) in (
-                child.children
-            ):  # find both X and Y in patterns of the form "the X and Y product"
-                if grandchild.dep_ == "conj":
+            for grandchild in child.children:
+                # find both X and Y in patterns of the form "the X and Y product"
+                if grandchild.dep_ == "conj" and not grandchild.is_space:
+                    # grandchild.is_space is handling double space cases
```
Comment (SanRehmo) [inline extra_model/_aspects.py:68]: inconsistent comments to line 47 and missing one in 49

File: setup.cfg [hunk 3]
```diff
@@ -20,7 +20,7 @@ classifiers =
 python_requires ~=3.8,<3.9
 install_requires =
     numpy==1.20.2
-    nltk==3.5.0
+    nltk==3.6.1
```
Comment (SanRehmo) [inline setup.cfg:23]: why was this bumped?

**Review by SanRehmo — CHANGES_REQUESTED** [review]
Please address the duplicate code in `_aspects.py`. Other than that, LGTM.

File: setup.cfg (hunk 3 above)
Comment (romatik) [reply setup.cfg:23]: Dependabot keeps all of the dependencies up-to-date, but it only updates `requirements.txt` automatically. I forgot to update `setup.cfg` in the PR (https://github.com/wayfair-incubator/extra-model/pull/85), so updated it here

Comment (SanRehmo) [reply setup.cfg:23]: 👍

File: extra_model/_aspects.py (hunk 1 above)
Comment (SanRehmo) [reply extra_model/_aspects.py]: Fine by me, but we should at least keep track of this and implement eventually. Do you have a place to keep track of such things?

**Review by SanRehmo — APPROVED** [review]

**Pull request merged**